webviz_config_equinor/assets/equinor_theme.css

```diff
@@ -41,3 +41,48 @@ div.styledLogo.tab {
     box-sizing: none;
     padding: 20px 30px;
 }
+
+/* 
+    Equinor branded table 
+
+    As of dash-table 3.6.0, styles are made inline. Therefore the use
+    of !important to override dash-table default style with Equinor style
+
+    When inline styles are removed from dash data-table, the !important
+    statement can be removed.
+*/
+
+table {
+    color: rgb(51, 51, 51);
+    border: solid white;
+    border-collapse: collapse;
+}
+
+td {
+    padding: 10px !important;
```

Comment:
The codacy errors was solved in webviz-config by adding a bandit.yml
https://github.com/equinor/webviz-config/blob/master/bandit.yml

Alternatively there might be a setting in codacy for it.